### PR TITLE
Feature/31

### PR DESCRIPTION
**Pull request by sglover**

### CODE REVIEW

File: helm/os-vim-driver/values.yaml [hunk 1]
```diff
@@ -44,7 +44,20 @@ app:
...
+      ssl:
+        enabled: True
+        secret:
+          ## Name of the secret containing the SSL certificate for the non-host based access (the CN of the certificate should match commonName)
+          name: ovd-tls
+          ## If True, the Helm chart installation will generate a new SSL key with a self-signed certificate
+          generate: True
+          ## The Common Name used for the SSL certificate
+          ## The default value here is acceptable but will not produce a completely valid certificate
+          ## You should update this value to the IP address used to access the API on this rule
```
Comment (dvaccarosenna) [inline helm/os-vim-driver/values.yaml]: Is this true? This value is correct as long as onboard the driver with the internal address "os-vim-driver" which is the main use case?

File: docker/Dockerfile [hunk 2]
```diff
@@ -2,6 +2,8 @@ FROM python:3.7-alpine
 
 ARG DRIVER_PORT
 ENV DRIVER_PORT ${DRIVER_PORT:-8292}
+ARG SSL_DRIVER_PORT
+ENV SSL_DRIVER_PORT ${SSL_DRIVER_PORT:-8392}
```
Comment (dvaccarosenna) [inline docker/Dockerfile]: Do we need an alternative port for SSL? Can we just use the one port?

File: helm/os-vim-driver/values.yaml (hunk 1 above)
Comment (sglover) [reply helm/os-vim-driver/values.yaml]: ok

File: docker/Dockerfile (hunk 2 above)
Comment (sglover) [reply docker/Dockerfile]: yep

**Pull request merged**